jmix-flowui/flowui/src/main/java/io/jmix/flowui/component/tabsheet/JmixTabSheet.java

```diff
@@ -373,6 +385,19 @@ public Object getSubPart(String name) {
                 .orElse(null);
     }
 
+    /**
+     * Add a new lazy tab to the component.
+     *
+     * @param lazyTab tab which components should be loaded after opening
+     * @param componentLoader tab loader
+     */
+    @Internal
+    public void addLazyTab(Tab lazyTab, ComponentLoader<?> componentLoader) {
+        getLazyTabs().add(lazyTab);
+
+        addSelectedChangeListener(new LazyTabChangeListener(componentLoader, lazyTab));
```

Comment:
1. `LazyTabChangeListener` cannot be overridden, so it's required to override the whole method.
2. Creating a separate listener for each lazy tab looks redundant. The whole class can be converted into method reference, since we don't need to pass Tab to check if it lazy.